share/completion/man

```diff
@@ -20,6 +20,7 @@ function completion/man {
         "a ${long:+--all}; show all manual pages, not only the first found one"
         "f ${long:+--whatis}; only print brief description"
         "k ${long:+--apropos}; search manual database and print brief description"
+        "l ${long:+--mandoc}; display man page given as filename"
```

Comment:
According to [man-db's manpage](https://gitlab.com/man-db/man-db/-/blob/2b38f74972ab09985b0f7a6716a08c657ff716d8/man/man1/man.man1#L484), the long version for `-l` is `--local-file`.
Is `--mandoc` for another implementation of `man` I don't know of?

```suggestion
        "l ${long:+--local-file}; display man page given as filename"
```